### DISCUSSION TITLE
Increase/Decrease buttons for changing the quantity - Inchoo extension

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#3491] [Extensions & Modules]
add quantity inchoo

as some of you know i'm trying to create a frontend theme very close to the standard magento 2 theme and at the same time improve its usability for mobuile devices (tablet, smart-phone)

I called this theme Mimic and made it available to the community.

In an attempt to evolve this theme I added a free plug-in by INCHOO that adds the "add quantity" function in the product page and in the minicart.

the plugin worked.

Since I updated open mage to version 20.1.0-rc6 this plug-in doesn't work anymore or rather it doesn't appear in the front-end anymore.0-rc6 questo plug-in non funziona più o meglio non compare più in front-end.

[Inchoo_Qty.zip](https://github.com/OpenMage/magento-lts/files/12511709/Inchoo_Qty.zip)

**Comment by fballiano** [comment]
it works for me on our `main` branch, also the javascript work, although it looks wrong

<img width="426" alt="Screenshot 2023-09-04 alle 12 29 24" src="https://github.com/OpenMage/magento-lts/assets/909743/2059af37-b4ed-4d94-8fad-003bbc013fd6">

**Comment by Danieleeffe1** [comment]
ok I solved it was a problem loading the css file associated with the buttons

below my modifications of the javascript.phtml file and the qty.css file for a correct display
![Screenshot_20230904_201732](https://github.com/OpenMage/magento-lts/assets/46442854/a3b7988e-37ce-48a1-9f31-7aaadf2fd453)
[qty.zip](https://github.com/OpenMage/magento-lts/files/12515875/qty.zip)
[javascript.phtml.zip](https://github.com/OpenMage/magento-lts/files/12515878/javascript.phtml.zip)
